### PR TITLE
5 ui5 plugin generation failing

### LINKED ISSUE #5: UI5 plugin generation failing
Issue opened by @rlopmar at https://github.com/SAP/generator-easy-ui5/issues/146

## Easy-Ui5 version: 

Node.js: v18.17.1
yeoman-environment: 3.19.3
generator-easy-ui5: 3.8.1

Available generators:
  - ui5-wdi5: 1.0.0
  - ui5-project: 0.3.1
  - ui5-flp-plugin: 0.0.1
  - ui5-app: 1.1.0

## OS/version:
Apple M2 Pro
iOS Version 14.5 

## Browser/version (+device/version):
NA - console execution

## Steps to reproduce the problem:

1. Step 1
Start easy-ui5 generator in console
> yo easy-ui5
2. Step 2
Select options in the command prompt as in the screenshot
3. Step 3
generator fails as show in the screenshot and code is not generated

Screenshot 2024-08-20 at 08 21 33

## What is the expected result?
Code generated without failing

## What happens instead?
No code generated

## Any other information? (attach screenshot if possible)
Screenshot above

### PR DESCRIPTION
**Pull request by ubheamar**

### CODE REVIEW

File: generators/app/index.js [hunk 1]
```diff
@@ -36,182 +52,158 @@ module.exports = class extends Generator {
...
                         "Add buttons to Me Area"
                     ];
                 },
-                default:[]
+                default: []
+            },
+            {
+                type: "input",
+                name: "author",
+                message: "Who is the author of the application?",
```
Comment (akudev) [inline generators/app/index.js:120]: Is "application" the right term for an FLP plugin?

File: generators/app/templates/readme.md [hunk 2]
```diff
@@ -1,7 +1,87 @@
...
+# UI5 Application <%= namespace %>
 
-Insert the purpose of this project and some interesting infos here
+Insert the purpose of this project and some interesting info here...
 
-## Credits
+## Description
 
-This project has been generated with 💙 and [easy-ui5](https://github.com/SAP)
+This app demonstrates a setup for developing UI5 applications.
```
Comment (akudev) [inline generators/app/templates/readme.md:7]: I don't think this fits for FLP plugins, does it?

File: generators/app/templates/webapp/flpSandbox.html [hunk 3]
```diff
@@ -49,46 +51,26 @@
...
-
-                // clean fiori loading screen markup before placing main content
-                var oContent = sap.ushell.Container.createRenderer();
-                jQuery("#canvas").empty();
-                oContent.placeAt("canvas");
-            }
-        </script>
     </head>
 
-    <body class="sapUiBody sapUShellFullHeight">
```
Comment (akudev) [inline generators/app/templates/webapp/flpSandbox.html:91]: the removal of these classes is intentional?

**Review by akudev — COMMENTED** [review]
It does look like a great overhaul and update.
I'm not too much an expert on the content, but at first glance it looks good to me.
Only at a few places I had the impression there were some copy&paste remains of an application template.

File: generators/app/index.js (hunk 1 above)
Comment (ubheamar) [reply generators/app/index.js]: changed it to FLP Plugin

File: generators/app/templates/readme.md (hunk 2 above)
Comment (ubheamar) [reply generators/app/templates/readme.md]: changed it to FLP Plugin

File: generators/app/templates/webapp/flpSandbox.html (hunk 3 above)
Comment (ubheamar) [reply generators/app/templates/webapp/flpSandbox.html:91]: yes it not required now

**Comment by petermuessig** [comment]
LGTM - sorry for the late review

**Pull request merged**